tests/behat/features/bootstrap/ApiFeatureContext.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace BehatTests;
+
+use Behat\Behat\Context\Context;
+use Behat\MinkExtension\Context\MinkContext;
+use GuzzleHttp\Client;
+
+/**
+ * Defines api features from the specific context.
+ */
+class ApiFeatureContext extends MinkContext implements Context {
+
+  /**
+   * @var Client
+   */
+  private $client;
+
+  /**
+   * @var string
+   */
+  private $actualPage;
+
+  /**
+   * ApiFeatureContext constructor.
+   */
+  public function __construct() {
+    $this->client = new Client(['base_uri' => $this->getMinkParameter('base_url')]);
+  }
+
+  /**
+   * @When /^I do a ([^"]*) request to "([^"]*)"$/
+   */
+  public function doRequest($method, $url) {
+    $uri = $this->getMinkParameter('base_url') . $url;
+    $request = $this->client
+      ->request($method, $uri, ['query' => ['_format' => 'json']]);
+    $this->actualPage = $request->getBody();
+  }
+
+  /**
+   * @Then /^I should find in the position ([^"]*) of the menu the "([^"]*)" with the value "([^"]*)"$/
+   * @throws \Exception
```

Comment:
@Saphyel I don't think this is generic enough, i.e. this test should have nothing do with a menu, but only with a REST response. Could we refactor this to reflect that, i.e. to say "Then the value X should be present in the response JSON".

Reply:
hmm...
You expect a valid JSON with the right format, if you only look for "X value is there" could be an issue for the Front End, so I disagree with a generic function

Reply:
@Saphyel what is the mentioning of "menu" used for? This isn't menu-specific, we'll have to assert JSON output in non-menu related JSON as well. My advice is to start small, i.e. "this text is available in that json" and then maybe add in positioning and depth in the array.

Reply:
We're looking at adding a general 'text is in the response' step definition over in https://github.com/comicrelief/behat-tools/pull/21

However I think there's a good case for next adding a more specific method in `behat-tools` too, that's still general enough to work for any similar JSON body. Something along the lines of `Some object in the root JSON array has _X_ property with _Y_ value`.

This would encourage implicit JSON validation when the 'is JSON' step isn't used, and the assertions can be more specific for any property check like this on any similar endpoint.